README.md

```diff
@@ -1,9 +1,14 @@
 <div align="center">
   <br>
-  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="Open Sauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png">
   <h1>🍕 Pizza CLI 🍕</h1>
-  <strong>A command line interface and tool for all things OpenSauced!</strong>
-  <br>
+  <strong>A Go command line interface for managing code ownership and project insights with OpenSauced!</strong>
+  <br/>
+   <a href="https://youtu.be/yINL-uIvhn0" target="_blank">
+    <img alt="CODEOWNERS demo" src="https://img.youtube.com/vi/yINL-uIvhn0/0.jpg" style="max-width:100%; cursor:pointer;margin-top: 20px;">
+  </a>
```

Comment:
I don't think this should be a YouTube video, but rather, a direct MP4 upload so people can play it right from the README. You can do this by dragging and dropping the mp4 into the text box when editing on GitHub

Reply:
@jpmcb This video is too big to do that. We need to get it down to 100MB or less. Right now it's 294.2.

Reply:
Oh wow, I didn't realize GitHub had that requirement: that makes sense.